src/foam/comics/DAOControllerView.js

```diff
@@ -212,6 +213,15 @@ foam.CLASS({
           .end());
 
       this.add(this.data.border);
+      if (this.isIframe()) this.tag(this.IFrameTopNavigation);
```

Comment:
spacing:
if ( this.isIframe() ) this.tag(this.IFrameTopNavigation);
Please read the FOAM style guide: https://github.com/foam-framework/foam/wiki/CodingStyle